### DISCUSSION TITLE
Option to view address on the same page as QR-code for validation

### DISCUSSION THREAD

**Discussion by Transisto** [btcpayserver/btcpayserver#4044] [Ideas / Feature Requests]
When serving an invoice as an in-person point of sale some people don't trust the QR-code and want to cross-check with the written address.

Example:

![image](https://user-images.githubusercontent.com/388716/184797668-b9765dfd-0679-496b-9158-207a257fd00b.png)

**Comment by Transisto** [comment]
In that POS scenario there is also no need for "Scan / Copy" buttons and "Open in wallet"

**Comment by Kukks** [comment]
Cc @dstrukt

**Reply by dennisreimann** [reply]
Yeah, good consideration for the checkout redesign!
